### PR TITLE
Feat/ibc whitelist

### PR DESCRIPTION
**Pull request by fragwuerdig**
## Summary of changes

This PR will whitelist all IBC denoms from burntax in the ante handler as requested by the community by passing proposal 11267.

## Report of required housekeeping

- [x] Github issue OR spec proposal link
- [x] Wrote tests
- [ ] Updated API documentation (client/lcd/swagger-ui/swagger.yaml)
- [ ] Added a relevant changelog entry: clog add [section] [stanza] [message]

----

## (FOR ADMIN) Before merging

- [ ] Added appropriate labels to PR
- [ ] Squashed all commits, uses message "Merge pull request #XYZ: [title]" (coding standards)
- [ ] Confirm added tests are consistent with the intended behavior of changes
- [ ] Ensure all tests pass

### CODE REVIEW

**Comment by fragwuerdig** [comment]
@nghuyenthevinh2000 - can you help me here?

I am trying to merge main into that branch so that we can approve on the pull request. But the Binance whitelist test in burntax_test.go does not go through the compiler - that's why the unit tests fail.

**Review by ZaradarBH — APPROVED** [review]

**Comment by ZaradarBH** [comment]
# github.com/terra-money/core/custom/auth/ante_test [github.com/terra-money/core/custom/auth/ante.test]
Error: custom/auth/ante/burntax_test.go:100:63: not enough arguments in call to ante.NewBurnTaxFeeDecorator

**Comment by ZaradarBH** [comment]
> @nghuyenthevinh2000 - can you help me here?
> 
> I am trying to merge main into that branch so that we can approve on the pull request. But the Binance whitelist test in burntax_test.go does not go through the compiler - that's why the unit tests fail.

I have fixed the broken test. It was a simple matter of merging the code from main onto this branch. Probably due to long lived branches :) https://medium.com/@peter.hozak/long-lived-git-branches-survival-guide-f3b1028d21d

**Review by inon-man — APPROVED** [review]

File: types/alias.go [hunk 1]
```diff
@@ -40,6 +40,7 @@ const (
 	BombayChainID         = "bombay-12"
 	SwapDisableForkHeight = fork.SwapDisableForkHeight
 	SwapEnableForkHeight  = fork.SwapEnableForkHeight
+	OsmoIbcDenom          = "ibc/0ef15df2f02480ade0bb6e85d9ebb5daea2836d3860e9f97f9aade4f57a31aa0"
```
Comment (nghuyenthevinh2000) [inline types/alias.go:43]: I just saw this variable OsmoIbcDenom, is it not used some where?

Comment (fragwuerdig) [reply types/alias.go:43]: It is used in tests to test against the regex.

**Review by fragwuerdig — COMMENTED** [review]
@edk208 what block height do we wanna use for this? Is 1st of march good?

**Pull request merged**